exporter/splunkhecexporter/README.md

```diff
@@ -145,11 +144,10 @@ exporters:
         custom_key: custom_value
 ```
 
-The full list of settings exposed for this exporter are documented [here](config.go)
-with detailed sample configurations [here](testdata/config.yaml).
+The full list of settings exposed for this exporter are documented in [config.go](./config.go)
+with detailed sample configurations in [testdata/config.yaml](./testdata/config.yaml).
 
-This exporter also offers proxy support as documented
-[here](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter#proxy-support).
+This exporter also [offers proxy support](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter#proxy-support).
```

Comment:
```suggestion
This exporter also offers [proxy support](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter#proxy-support).
```
Nit

Reply:
Sigh, looks like I _mostly_ settled on that (note that the presence of the anchor unlike queued-retry which doesn't have it makes this correct).